modules/ovis/include/opencv2/ovis.hpp

```diff
@@ -164,6 +164,25 @@ class CV_EXPORTS_W WindowScene {
     CV_WRAP virtual void setEntityPose(const String& name, InputArray tvec = noArray(),
                                        InputArray rot = noArray(), bool invert = false) = 0;
 
+    /**
+     * update entity animation state
+     * @param name enitity name
+     * @param animname animation state name
+     * @param offset amount of time, in seconds, to extend the animation
+     */
+    CV_WRAP virtual void updateEntityAnimation(const String& name, const String& animname,
+                                               double offset) = 0;
+
+    /**
+     * set entity animation state
+     * @param name enitity name
+     * @param animname animation state name
+     * @param enable activate or deactivate animation
+     * @param loop activate or deactivate animation loop
+     */
+    CV_WRAP virtual void setEntityAnimation(const String& name, const String& animname,
```

Comment:
change this to `void enableEnitityAnimation(str name, str animname, bool)` implementing the behavior outlined above.
Move the loop flag to the `cv::ovis::EntityProperty` enum as `ENTITY_ANIMATION_LOOP`

Reply:
Thanks, but this one I couldn't agree on: An entity can carry multiple animations and you want to be able to tell which one of them should be played in a loop and which one not. Having a property on the entity would restrict us to a global behavior.

Instead to changing this to `enableEntityAnimation` I changed it to `playEntityAnimation`, which by default plays / enables the animation state. The only flag left to set is whether to loop the animation or not.

Another method `stopEntityAnimation` has been added to stop the animation playback and clean up, similar to the reset function in the _ogre mesh viewer_.